.github/workflows/pull.yml

```diff
@@ -0,0 +1,27 @@
+name: Pull Request Checks
+
+on:
+  pull_request:
+
+jobs:
```

Comment:
Does this also run the python-examples and python-integration so they get the "required" treatment?